requirements.yml

```diff
@@ -4,22 +4,24 @@
   version: 1.1.0
 
 - src: ome.ice
-  version: 3.0.0
+  version: 3.1.0
 
 - src: ome.java
   version: 2.0.3
 
-- src: ome.omego
-  version: 0.1.3
+- name: ome.omego
+  src: https://github.com/joshmoore/ansible-role-omego
```

Comment:
Proposing to update this personal branch before we create a Docker image? I think either cutting a release of `ome.omego` with https://github.com/ome/ansible-role-omego/pull/6 or testing a more recent version of https://github.com/ome/ansible-role-omero-server/pull/40 with `omego` is installed in the virtualenv would be options.

Reply:
I'm currently trying the latter on the omero-web-docker PR.